wolnut/__init__.py

```diff
@@ -1,7 +1,18 @@
-import logging
+__title__ = 'wolnut'
+__author__ = 'hardwarehaven'
+__license__ = 'MIT'
+__copyright__ = 'Copyright 2025 hardwarehaven'
+__version__ = '1.0.0'
 
-logging.basicConfig(
-    level=logging.INFO,
-    format="%(asctime)s [%(levelname)s] %(message)s",
-)
-logger = logging.getLogger("wolnut")
+
+def entrypoint():
+    import logging
+
+    from .__main__ import main
```

Comment:
I don't know if you need this. That's the point of the `__main__.py`. https://docs.python.org/3/library/__main__.html

`python -m wolnut` will run `__main__.py` directly.

Reply:
I did not know you can add plain files as the entrypoint, thought you had to import a function. Will try and update if necessary.

Reply:
Just tested it. You can't. So entrypoint is needed to setup logging.